app/DataTables/SensorDataDataTable.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\DataTables;
+
+use Yajra\DataTables\Services\DataTable;
+use App\SensorData;
+
+class SensorDataDataTable extends DataTable
+{
+    /**
+     * Build DataTable class.
+     *
+     * @return \Yajra\DataTables\Engines\BaseEngine
+     */
+    public function dataTable($query)
+    {
+        return datatables($query)
+            ->editColumn('sensor_id', function ($sensordata) {
+                return '<a href="/sensor/' . $sensordata->sensor_id . '">'. ($sensordata->sensor->name ?? '') . '</a>';
+            })
+            ->addColumn('action', 'sensordata.action')
+            ->blacklist([ 'action'])
+            ->rawColumns(['sensor_id', 'action']);
+    }
+
+    /**
+     * Get the query object to be processed by dataTables.
+     *
+     * @return \Illuminate\Database\Eloquent\Builder|\Illuminate\Database\Query\Builder|\Illuminate\Support\Collection
+     */
+    public function query()
+    {
+        $query = SensorData::query();
+        return $this->applyScopes($query);
+    }
+
+    /**
+     * Optional method if you want to use html builder.
+     *
+     * @return \Yajra\DataTables\Html\Builder
+     */
+    public function html()
+    {
+        return $this->builder()
+            ->columns($this->getColumns())
+            ->minifiedAjax()
+            ->parameters($this->getBuilderParameters());
+    }
+
+    /**
+     * Get columns.
+     *
+     * @return array
+     */
+    protected function getColumns()
+    {
+        return [
+            'id',
+            'sensor_id',
+            'value',
+            'created_at',
+            'updated_at',
+            'action'
```

Comment:
The action column shouldn't be orderable.

Reply:
fixed in 86de04d